.github/workflows/rotki_sqldiff.yml

```diff
@@ -7,6 +7,10 @@ on:
       - bugfixes
       - master
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
github.ref in the case of this PR is `kelsos:add_concurrency_option` right?

Reply:
I might need to add head let me check the action